test/test_jpmge.jl

```diff
@@ -126,6 +132,264 @@
     @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[l]†X[s2]")]) ≈ 3.06483479678299# not quite…3.06483456
     @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[k]†X[s2]")]) ≈ 0.949034056889784# not quite…0.94903402
     @test MPSGE.Complementarity.result_value(m._jump_model[:Y]) ≈ 6.55295034252603# not quite…6.55295028
-    @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("P[g1]ρY")]) ≈ 2.10582458311357# not quite…2.10582449
-    @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("P[g2]ρY")]) ≈ 4.28949139501026# not quite…4.28949135                                                     
+    @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("P[g1]ρY")]) ≈ two_by_two_jpmge["PY.g1","PF(\"labor\")=1"] # 2.10582458311357# not quite…2.10582449
+    @test MPSGE.Complementarity.result_value(m._jump_model[Symbol("P[g2]ρY")]) ≈ two_by_two_jpmge["PY.g2","PF(\"labor\")=1"] # 4.28949139501026# not quite…4.28949135                                                     
+
+
+#A replication of the JPMGE from https://www.gams.com/34/docs/UG_MPSGE_Intro.html#UG_MPSGE_Intro_Appendix_jpmge
```

Comment:
Indentation is wrong, and put this into a new file as a new testset, much easier to handle that way.